subprojects/docs/src/docs/release/notes.md

```diff
@@ -76,67 +76,91 @@ vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv -->
 ```
-TestLauncher testLauncher = projectConnection.newTestLauncher();
-testLauncher.withTestsFor(spec -> {
-    spec.forTaskPath(":test")
-        .includePackage("org.pkg")
-        .includeClass("com.TestClass")
-        .includeMethod("com.TestClass")
-        .includePattern("io.*")
-}).run();
+
+In your build scripts or convention plugins:
+```kotlin
+tasks.wrapper {
+    networkTimeout.set(30000)
+}
 ```
 
-Note, that the new test selection interface only works if the target Gradle version is >=7.6.
+Or in `gradle/wrapper/gradle-wrapper.properties`:
+```properties
+networkTimeout=30000
+```
 
-### Improvements to the `init` plugin
+See the [user manual](userguide/gradle_wrapper.html#sec:adding_wrapper) for more information.
 
-#### Use included build for convention plugins (incubating)
 
-When generating builds using the `init` task and opting in to using incubating features, convention plugins are now located in an included build under the `gradle/plugins` directory instead of in `buildSrc`.
+#### Introduced flag for individual task `rerun` 
+
+Every task can now use the `--rerun` option. This option works like `--rerun-tasks`,
+except `--rerun` only effects a single task. For example, you can force tests to
+ignore up-to-date checks like this:
+```
+gradle test --rerun
+```
 
-#### Improved Maven Conversion
+See the [documentation](userguide/command_line_interface.html#sec:builtin_task_options) for more information.
 
-The `init` task now adds compile-time Maven dependencies to Gradle's `api` configuration when converting a Maven project. This sharply reduces the number of compilation errors resulting from the automatic conversion utility. See the [Build Init Plugin](userguide/build_init_plugin.html#sec:pom_maven_conversion) for more information.
 
-<a name="configuration-cache-improvements"></a>
-### Configuration cache improvements
+<a name="configuration"></a>
+### Configuration
 
 The [configuration cache](userguide/configuration_cache.html) improves build time by caching the result of the configuration phase and reusing this for subsequent builds.
 
-#### New compatible tasks
+#### Extended configuration cache task compatibility
 
 The `dependencies`, `buildEnvironment`, `projects` and `properties` tasks are now compatible with the configuration cache.
 
-### Resolvable configurations report displays attribute precedence order
 
-The `resolvableConfigurations` reporting task will now print the order that [attribute disambiguation rules](userguide/variant_attributes.html#sec:abm_disambiguation_rules) will be checked when resolving project dependencies.  These rules are used if multiple variants of a dependency are available with different compatible values for a requested attribute, and no exact match.  Disambiguation rules will be run on all attributes with multiple compatible matches in this order to select a single matching variant.
+#### Clarified attribute ordering for `resolvableConfigurations` reports  
```

Comment:
This isn't clarifying the attribute ordering, that remains the same and is printed the same (alphabetically) in this report.  It's clarifying the order that disambiguation rules will be run if they are needed.  Perhaps:

"Clarified the order disambiguation rules will be checked when multiple compatible attributes exist in `resolvableConfigurations` report"

That's also a mouthful, but there is a very specific use case when this will be helpful.

Reply:
How about:

```suggestion
#### Clarified the ordering of disambiguation rule checks in `resolvableConfigurations` reports  
```
Slightly less of a mouthful, but I'm not 100% sure if this is true. I get the sense that "when multiple compatible attributes exist" is the only time when this *matters*, but technically the disambiguation rules apply in all cases.

Would this reword be OK? Or should I use the longer rewording that you suggested above?
@tresat 

Thanks!

Reply:
That's correct, and sounds fine to me.